Pipfile

```diff
@@ -10,6 +10,11 @@ pre-commit = "*"
 coverage = "*"
 pytest-cov = "*"
 codacy-coverage = "*"
+iolite = {editable = true,path = "."}
```

Comment:
For me, the example.py threw an error because iolite was not an installed package. So I added the iolite directory as an editable dev-dependency. This way you can `import iolite` without having the package actually installed in the environment.

Reply:
Ahh now I see it. It feels weird having the package add itself as a dependency but I am not sure of a better way.

Reply:
Another solution would be to turn `scripts` into a module (add `__init__.py`) and then it can be invoked as `python -m scripts.example`

Reply:
That would work for the scripts, yes. I just realized the `iolite = {editable = true,path = "."}` is also needed to import iolite inside the notebooks, though. So I would like to keep this change.